Smartpad-macOS/Smartpad-macOS/Logic/ConnectionManager.swift

```diff
@@ -99,12 +155,39 @@ extension ConnectionManager{
             GestureGenerator.getGesture(type: packet.touchType)
                             .performGesture(packet: packet)
     }
+
     
-    func browserViewControllerDidFinish(_ browserViewController: MCBrowserViewController) {
-        pairVC!.dismiss(browserViewController)
+    // Callbacks for the peer browser
+    
+    func browser(_ browser: MCNearbyServiceBrowser, foundPeer peerID: MCPeerID, withDiscoveryInfo info: [String : String]?) {
+
+        peerList.append(peerID)
```

Comment:
I wonder if there is a state that determines append/remove? This might be why I am seeing ghost devices? We add peerID here, but where is peerID ever removed if the device goes out of range while in the peer menu?

Reply:
It's weird that it's showing multiple devices. I only see 1.

Reply:
nvm. I'm seeing the bug.

Reply:
Actually, I'm able to see this issue on the main branch as well, so we should merge this change first then fix that issue in another PR